### PR TITLE
AP_Camera: increase camera triggers distance accuracy

### PR DESCRIPTION
**Pull request by mirkix**
This PR increase the camera trigger accuracy when using automatic camera trigger with https://ardupilot.org/copter/docs/parameters.html#cam-trigg-dist-camera-trigger-distance. Until now a new trigger will be checked with each GPS update (~10Hz), after this PR the trigger will  be checked with 50Hz.
With ~10Hz and flight speed of 15m/s we only check about each 1,5m if we have to trigger the camera again, this should be improved.

### CODE REVIEW

**Comment by mirkix** [comment]
@tridge After our short discussion on discord about that, do you think this could be a good solution to increase the accuracy?

File: ArduCopter/Copter.cpp [hunk 1]
```diff
@@ -514,25 +514,7 @@ void Copter::one_hz_loop()
 // called at 50hz
 void Copter::update_GPS(void)
```
Comment (WickedShell) [inline ArduCopter/Copter.cpp:515]: If this is the change set, shouldn't this whole function be removed and instead replace the scheduler entry with:

`SCHED_TASK_CLASS(AP_GPS,      &Copter.gps,  update, 50, 300),`

Comment (mirkix) [reply ArduCopter/Copter.cpp]: @WickedShell Yes, that makes sense.

File: ArduCopter/Copter.cpp [hunk 2]
```diff
@@ -511,30 +511,6 @@ void Copter::one_hz_loop()
     AP_Notify::flags.flying = !ap.land_complete;
 }
 
-// called at 50hz
-void Copter::update_GPS(void)
```
Comment (WickedShell) [inline ArduCopter/Copter.cpp:515]: Don't forget to remove the header entry for these.

Comment (mirkix) [reply ArduCopter/Copter.cpp:515]: @WickedShell Done

**Comment by mirkix** [comment]
@WickedShell Should we add reviewers for Rover, Plane, Copter and Sub?

**Pull request closed without merging**